docusaurus-docs/jupyterlab-conda-store/introduction.md

```diff
@@ -1,7 +1,71 @@
 ---
-sidebar_position: 1
 title: Introduction
 description: Introduction to JupyterLab Extension.
 ---
 
 # conda-store JupyterLab extension
+
+A extension to use the [conda-store UI][conda-store-ui] - a React-based frontend for conda-store, within JupyterLab.
+
+## Install 📦
+
+1. Pre-requisites: `conda-store-server`, JupyterLab `>=` 3.0 and `<=` 4.0, and Python `>=` 3.8 installed.
```

Comment:
Is it necessary to put the comparison operators in backticks?

Reply:
Yes, with the more recent MDX versions, the `<, >` characters cause issues. Ref: https://docusaurus.io/blog/releases/3.0#markdown

Reply:
Oh right. I wonder if you can do could use HTML entities like `&gt;` (greater than) and `&lt;` (less than), which I believe is supported by GitHub flavored markdown so might be supported in these docs but I don't know for sure. But this is non-blocking ;)

Reply:
Oh nice, I'll update, thanks!